<launch>

	<node pkg="v2x_ros" type="v2x_client_v1" name="v2x_client_v1" output="screen" respawn="true" respawn_delay="1" >
		<param name="csv_file_path" value="$(find v2x_ros)/mission_list.csv" />
		<param name="obu_ip" value="172.20.0.1" />
		<param name="obu_port" value="17001" />
		<param name="buffer_size" value="1500" />
		<param name="sleep_rate" value="100" /> 
		<!-- V2X: "OFFLINE" or "ONLINE" -->
		<param name="v2x_env" value="ONLINE" /> 
		<!-- ROSBAG: "true" or "false" -->
		<param name="rosbag_env" value="true" /> 

		<param name="distance_threshold" value="1.0" />
		<param name="time_threshold" value="5.0" />
		
	</node>

</launch>
